### ISSUE TITLE
TileLayer Support

### ISSUE THREAD

**Issue by aarontimbrook** [infusion-code/angular-maps#62]
Any plans to include basic TileLayer support?

**Comment by METACEO** [comment]
There have certainly been discussions, yes - what interactions would you expect (or what would you require) out of such a feature?

**Comment by aarontimbrook** [comment]
Thanks for the quick response and consideration for this enhancement request.
I would like to be able to add a static weather layer, something like...

const url = `http://idpgis.ncep.noaa.gov/arcgis/services/NWS_Observations/radar_base_reflectivity/MapServer/WmsServer?REQUEST=GetMap&SERVICE=WMS&VERSION=1.3.0&LAYERS=1&STYLES=default&FORMAT=image/png&TRANSPARENT=TRUE&CRS=CRS:84&BBOX={bbox}&WIDTH=256&HEIGHT=256`;
const sourceOptions: Microsoft.Maps.ITileSourceOptions = {
  uriConstructor: url,
  minZoom: 3,
  maxZoom: 10
};
const tileSource = new Microsoft.Maps.TileSource(sourceOptions);
const tileLayerOptions: Microsoft.Maps.ITileLayerOptions = {
  mercator: tileSource,
  opacity: 1.0,
  visible: true,
  zIndex: 9,
};
const tileLayer = new Microsoft.Maps.TileLayer(tileLayerOptions);
map.layers.insert(tileLayer);

Although an AnimatedTileLayer to show an animated weather layer for a timeframe would be great too.

**Comment by thor-schueler** [comment]
Yes, we are a looking to add this capability with a new component. Stay tuned. 

In the meanwhile you can probably do it manually (like you illustrate above) by getting a reference to the bing map object from the Map Service (you can inject that into your component).
